Add Nvidia.md leaf file, add links in sidebar & getting started (#2029)
YaLTeR/niri: wiki/Getting-Started.md

### NVIDIA

> The NVIDIA drivers currently have an issue with high VRAM usage due to a Heap reuse quirk. There is a manual fix documented [here](./Nvidia.md) that you _SHOULD_ apply to your system if you use an NVIDIA gpu and niri.

Reviewer: ```suggestion
The NVIDIA drivers currently have an issue with high VRAM usage due to a heap reuse quirk.
You're recommended to apply a manual fix documented [here](./Nvidia.md) if you run niri on an NVIDIA GPU.
```
